test/feature.go

```diff
@@ -50,17 +56,17 @@ func parseFeatures(filter string, paths []string) ([]*gherkin.Feature, error) {
 		}
 
 		for _, ft := range fts {
-			if _, duplicate := features[ft.Name]; duplicate {
+			if _, duplicate := features[ft.document.Feature.GetName()]; duplicate {
 				continue
 			}
-			features[ft.Name] = ft
+			features[ft.document.Feature.GetName()] = ft
 		}
 	}
 	return filterFeatures(filter, features), nil
 }
 
-func parsePath(path string) ([]*gherkin.Feature, error) {
-	var features []*gherkin.Feature
+func parsePath(path string) ([]*feature, error) {
```

Comment:
Should this method be renamed to parseFeaturesFromPath to make clear the purpose?

Reply:
as said, I just copied and did not much touch this code ...